### PR TITLE
Algo display

### PR DESCRIPTION
**Pull request by j2kan**
Changes:
1. Display the position each student received. It looks for lowest score to assign full ta first.
2. Display students who are not matched and export functionality for it.
3. Display courses that are not matched in terms of positions remaining and no longer aggregate and export functionality for it.
4. Added instructor to export and display.
5. fix uni-code error.
6. Added system and testing documentation

### CODE REVIEW

**Comment by j2kan** [comment]
@amyleblond @lauradobson @smwatts 
new incentive, find a valid bug in this PR. I buy you ice cream.

File: taworks/taform/templates/taform/algorithm.html [hunk 1]
```diff
@@ -31,29 +31,56 @@ <h1>Teaching Assistant Assignment</h1>
...
         </td>
     </tr>
     <tr>
-        <td>2. Export algorithm results.&emsp;</td>
+        <td>2. Export students without a position.&emsp;</td>
         <td>
         <form action="{% url 'algorithm' %}" method="post">
         {% csrf_token %}
-            <input type="submit" name="algo_export" value="Export"/>
+            <input class="submitButton2" type="submit" name="student_export" value="Export students w/o match"/>
```
Comment (lauradobson) [inline taworks/taform/templates/taform/algorithm.html]: Can we make the buttons say "Students without a match" and "Courses without a match"?

**Review by amyleblond — APPROVED** [review]
you know whats up

also byebye Barney

**Review by amyleblond — DISMISSED** [review]
you know whats up

and byebye Barney

**Comment by j2kan** [comment]
@amyleblond fixed barney and made font smaller and fixed bug you found, IOU froyo
@lauradobson changed wording

**Review by amyleblond — APPROVED** [review]
this one runs! no issues

**Review by smwatts — APPROVED** [review]
Tested and works amazing!

**Comment by smwatts** [comment]
If there are no "unmatched" students or "unmatched" courses do we want to explicitly say "no unmatched students/courses" or just not have the tables display? Right now the tables dont display :)

**Pull request merged**